app/src/main/java/mozilla/lockbox/autofill/FillResponseBuilder.kt

```diff
@@ -100,6 +104,7 @@ class FillResponseBuilder(
                 AutofillValue.forText(credential.password), passwordPresentation)
         }
 
+        // is this building a concatinated length of autofill_items?
```

Comment:
`RemoteViews` representation == a username, or a password.
`Dataset` == one username/password combination
`FillResponse` == a collection of datasets.